test/functional/crud_spec.test.js

```diff
@@ -433,3 +436,29 @@ describe('CRUD v2', function () {
 
   generateTopologyTests(testSuites, testContext);
 });
+
+describe('CRUD unified', function () {
+  for (const crudSpecTest of loadSpecTests('crud/unified')) {
+    expect(crudSpecTest).to.exist;
+    context(String(crudSpecTest.description), function () {
+      for (const test of crudSpecTest.tests) {
+        it(String(test.description), {
+          metadata: { sessions: { skipLeakTests: true } },
+          test: async function () {
+            try {
+              await runUnifiedTest(this, crudSpecTest, test);
+            } catch (error) {
+              if (error.message.includes('not implemented.')) {
+                console.log(`${test.description}: was skipped due to missing functionality`);
+                console.log(error.stack);
+                this.skip();
```

Comment:
There shouldn't be anymore 'not implemented.' errors in the runner I believe so we can drop this.